what4/src/What4/Protocol/SMTWriter.hs

```diff
@@ -867,12 +867,18 @@ class (SupportTermOps (Term h)) => SMTWriter h where
   --   later reporting unsatisfiable cores).
   assertNamedCommand :: f h -> Term h -> Text -> Command h
 
-  -- | Push 1 new scope
+  -- | Generates command `(push 1)` that opens the corresponding assertion frame
   pushCommand   :: f h -> Command h
 
-  -- | Pop 1 existing scope
+  -- | Generates command `(pop 1)` that closes the corresponding assertion frame
   popCommand    :: f h -> Command h
 
+  -- | Generates command `(push 2)` that opens the corresponding assertion frame
```

Comment:
A minor thing: Haddock uses the `@` character to delimit inline code, not a backtick:

```suggestion
  -- | Generates command @(push 2)@ that opens the corresponding assertion frame
```